webrtc-stats.html

```diff
@@ -1612,6 +1612,8 @@
         </h3>
         <div>
           <pre class="idl">dictionary RTCIceCandidatePairStats : RTCStats {
+             DOMHighResTimeStamp           lastPacketSentTimestamp;
+             DOMHighResTimeStamp           lastPacketReceivedTimestamp;
```

Comment:
Style: Suggest moving these counters below bytesSent and bytesReceived, so that we keep the counters together.